mediator/med_phases_restart_mod.F90

```diff
@@ -346,6 +346,10 @@ subroutine med_phases_restart_write(gcomp, rc)
              if (is_local%wrap%comp_present(n)) then
                 nx = is_local%wrap%nx(n)
                 ny = is_local%wrap%ny(n)
+                if (is_local%wrap%ntile(n) > 0) then
```

Comment:
Is this still writing restart files in 1d?

Reply:
Yes, it will still write restart files on the mesh.

Reply:
@DeniseWorthen Thanks.